spec/unit/sanitize_spec.rb

```diff
@@ -16,4 +16,12 @@
       expect(Strings::ANSI.sanitize(code)).to eq(expected)
     end
   end
+
+  it 'removes escape codes from string containing hyperlink escape codes' do
+    expect(Strings::ANSI.sanitize("\e]8;;https://google.com\a\e]8;;\a")).to eq("")
+    expect(Strings::ANSI.sanitize("\e]8;;https://google.com\e\\\e]8;;\e\\")).to eq("")
+
+    expect(Strings::ANSI.sanitize("\e]8;;https://google.com\alabel\e]8;;\a")).to eq("label")
+    expect(Strings::ANSI.sanitize("\e]8;;https://google.com\e\\label\e]8;;\e\\")).to eq("label")
+  end
```

Comment:
Could you please split these out into separate test cases to make it easier for future maintenance?